src/file_utils.cr

```diff
@@ -24,4 +24,29 @@ module FileUtils
       return true if read1 == 0
     end
   end
+
+  # Copies a file content *src* to *dest*. If dest is a directory, copies *src* to *dest/src*
```

Comment:
`file contents of *src_path*` might be less confusing. Also mentioning that the basename is taken in the second case might be worth it.